LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t04_lib.dart

```diff
@@ -7,9 +7,9 @@
 /// corresponding type in the augmented library. In other words, the
 /// augmentation can add new members to an existing type.
 ///
-/// @description Checks that static members defined in the body of an augment of
-/// a class, mixin or enum are added to a static namespace of the corresponding
-/// type in the augmented library
+/// @description Checks that instance members defined in the body of an augment
+/// of a class, mixin, extension, or enum are added to an instance namespace of
+/// the corresponding type in the augmented library
```

Comment:
Wouldn't that still be 'static'?
```suggestion
/// @description Checks that static members defined in the body of an augment
/// of a class, mixin, extension, or enum are added to the static namespace of
/// the corresponding type in the augmented library
```

Reply:
We could say that the augmented library contains the corresponding type in any case, semantically, but the actual declaration could also be in an augmentation library.